static/css/notfound.css

```diff
@@ -0,0 +1,96 @@
+* {
```

Comment:
if we're using bootstrap why the need to specify so much css? Shouldn't bootstrap handle that for us?